extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1031,6 +1049,52 @@ export class Project implements ISqlProject {
 			}
 		}
 	}
+
+	/**
+	 * Evaluates the value of the property item in the loaded project.
+	 *
+	 * @param propertyName Name of the property item to evaluate.
+	 * @returns Value of the property or `undefined`, if property is missing.
+	 */
+	private evaluateProjectPropertyValue(propertyName: string): string | undefined;
+
+	/**
+	 * Evaluates the value of the property item in the loaded project.
+	 *
+	 * @param propertyName Name of the property item to evaluate.
+	 * @param defaultValue Default value to return, if property is not set.
+	 * @returns Value of the property or `defaultValue`, if property is missing.
+	 */
+	private evaluateProjectPropertyValue(propertyName: string, defaultValue: string): string;
+
+	/**
+	 * Evaluates the value of the property item in the loaded project.
+	 *
+	 * @param propertyName Name of the property item to evaluate.
+	 * @param defaultValue Default value to return, if property is not set.
+	 * @returns Value of the property or `defaultValue`, if property is missing.
+	 */
+	private evaluateProjectPropertyValue(propertyName: string, defaultValue?: string): string | undefined {
+		// TODO: Currently we simply read the value of the first matching element. The code should be updated to:
+		//       1) Narrow it down to items under <PropertyGroup> only
+		//       2) Respect the `Condition` attribute on group and property itself
+		//       3) Evaluate any expressions within the property value
+
+		// Check if property is set in the project
+		const propertyElements = this.projFileXmlDoc.getElementsByTagName(propertyName);
+		if (propertyElements.length === 0) {
+			return defaultValue;
+		}
+
+		// Try to extract the value from the first matching element
+		const firstPropertyElement = propertyElements[0];
+		if (firstPropertyElement.childNodes.length !== 1) {
```

Comment:
This is the original logic and I didn't touch it, although I'm not sure if it's correct. This will still allow
```xml
<Prop1><foo/></Prop1>
```
which is probably not expected.